extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -206,6 +206,16 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
       },
       true /** useCapture */
     );
+
+    // Closes the remote attachment drawer when navigation deeplinked to an app.
+    if (this.type_ === AttachmentType.REMOTE) {
+      const ampdoc = this.getAmpDoc();
+      ampdoc.onVisibilityChanged(() => {
+        if (ampdoc.isVisible() && this.state_ === DrawerState.OPEN) {
```

Comment:
this works? do you know why is the ampdoc visible when navigating to a different app?

Reply:
Most code is paused while the tab/browser is inactive, so this actually checks when you're coming back. It reads: visibility changed to active (welcome back!) AND drawer is open -> let's close it